Author: Andreas Tille <tille@debian.org>,
        Svante Signell <svante.signell@gmail.com>
Last-Uudate: Wed, 31 Aug 2016 12:14:45 +0200
Bug-Debian: https://bugs.debian.org/793512
            https://bugs.debian.org/836190
Description: Define PATH_MAX as it is done in cram/open_trace_file.c if
 not existent.  The definition should be removed from this C file since the
 header file affects all three C files where this definition is used.

--- a/cram/os.h
+++ b/cram/os.h
@@ -160,6 +160,13 @@ static inline uint16_t le_int2(uint16_t
 }
 #endif
 
+/*
+ * in case PATH_MAX is not defined (like in hurd)
+ */
+#ifndef PATH_MAX
+#  define PATH_MAX 1024
+#endif
+
 /*-----------------------------------------------------------------------------
  * Operating system specifics.
  * These ought to be done by autoconf, but are legacy code.
